.github/actions/sil-kit-ci/cmake-configure-args/action.yml

```diff
@@ -0,0 +1,35 @@
+name: sil-kil-ci/cmake-configure-args
+
+description: Compute arguments passed to cmake to override or augment the presets.
+
+inputs:
+
+  linux-c-compiler:
+    description: C compiler used under Linux
+    required: true
+
+  linux-cxx-compiler:
```

Comment:
the MSVC tools are set in the setup. I wonder if we could merge the two, saving one sub-action, that is lift this code up into the setup action? Or am I missing something

Reply:
I like that, this is essentially a part of the setup.

Reply:
Done 👍 It does fit very well into the `setup` action. The output is now called `cmake-configure-args`.